README.md

```diff
@@ -45,71 +45,53 @@ from prefect import flow
 from prefect_dbt.cloud import DbtCloudCredentials
 from prefect_dbt.cloud.jobs import trigger_dbt_cloud_job_run_and_wait_for_completion
 
+
 @flow
-def run_dbt_job_flow():
+def run_dbt_cloud_job():
     run_result = trigger_dbt_cloud_job_run_and_wait_for_completion(
-        dbt_cloud_credentials=DbtCloudCredentials(
-            api_key="my_api_key",
-            account_id=123456789
-        ),
+        dbt_cloud_credentials=DbtCloudCredentials.load("dev"),
         job_id=1
     )
 
-run_dbt_job_flow()
+if __name__ == "__main__":
+    run_dbt_cloud_job()
 ```
 
 ### Execute a dbt CLI command
 ```python
 from prefect import flow
 from prefect_dbt.cli.commands import trigger_dbt_cli_command
 
+
 @flow
-def trigger_dbt_cli_command_flow() -> str:
+def execute_dbt_command() -> str:
     result = trigger_dbt_cli_command("dbt debug")
-    return result # Returns the last line the in CLI output
+    return result  # Returns the last line the in CLI output
 
-trigger_dbt_cli_command_flow()
+if __name__ == "__main__":
+    execute_dbt_command()
 ```
 
 ### Execute a dbt CLI command without a pre-populated profiles.yml
 ```python
 from prefect import flow
-from prefect_snowflake.credentials import SnowflakeCredentials
-from prefect_snowflake.database import SnowflakeConnector
 
 from prefect_dbt.cli.credentials import DbtCliProfile
 from prefect_dbt.cli.commands import trigger_dbt_cli_command
-from prefect_dbt.cli.configs import SnowflakeTargetConfigs
+
 
 @flow
-def trigger_dbt_cli_command_flow():
-    connector = SnowflakeConnector(
-        schema="public",
-        database="database",
-        warehouse="warehouse",
-        credentials=SnowflakeCredentials(
-            user="user",
-            password="password",
-            account="account.region.aws",
-            role="role",
-        ),
-    )
-    target_configs = SnowflakeTargetConfigs(
-        connector=connector
-    )
-    dbt_cli_profile = DbtCliProfile(
-        name="jaffle_shop",
-        target="dev",
-        target_configs=target_configs,
-    )
+def execute_dbt_command():
+    dbt_cli_profile = DbtCliProfile.load("dev")
```

Comment:
I'd like to split the different here and show the `SnowflakeConnetor` being loaded and then passed into `DbtCliProfile` to show different ways of instantiating blocks. The `DbtCliProfile` doesn't contain any secrets directly, so I think it's reasonable to create it inline.

Reply:
True, thanks, Alex. I'll show an example of how to separately create a Block -- both from code and the UI.

Reply:
This can demonstrate how to load blocks:
https://github.com/PrefectHQ/prefect-dbt/pull/66

Reply:
I think the cruft update can resolve this too:
https://github.com/PrefectHQ/prefect-dbt/pull/75/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R29